src/script/removeCompleted.js

```diff
@@ -0,0 +1,7 @@
+export default function taskCompleted() {
+  let tasks = JSON.parse(localStorage.getItem('tasks'));
+  const arr1 = tasks.filter((el) => el.completed === false);
+  console.log(arr1);
+  tasks = arr1;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
- [ ] Nice job :clap: on making this functionality however I can see you are not updating your ids(indexes) when cleaning completed tasks or deleting them, kindly let's go and make this last touch.

Reply:
Asked change implemented